site/frontend/src/pages/compare/compile/table/benchmark-detail.vue

```diff
@@ -359,16 +381,20 @@ onMounted(() => renderGraph());
     flex-grow: 1;
   }
 }
+.columns + .columns {
```

Comment:
I have been doing webdev for 12+ years and this is the first time I'm seeing this :eyes: Bloody CSS. I would probably just add a new class, like `.graphs`, and add the style to that, to make it more obvious/readable.

Reply:
Done